src/main/java/org/folio/service/consortium/ConsortiumConfigurationService.java

```diff
@@ -0,0 +1,42 @@
+package org.folio.service.consortium;
+
+import io.vertx.core.Future;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+import org.folio.models.consortium.ConsortiumConfiguration;
+import org.folio.rest.core.RestClient;
+import org.folio.rest.core.models.RequestContext;
+import org.folio.rest.core.models.RequestEntry;
+
+import java.util.Optional;
+
+public class ConsortiumConfigurationService {
```

Comment:
consider having a cache to keep these values. They do no change often.

Reply:
Thanks. Added